demos/openvino/run_benchmark_on_occlum.sh

```diff
@@ -9,25 +9,39 @@ rm -rf occlum_instance
 mkdir occlum_instance
 cd occlum_instance
 occlum init
+cpu_cc=`cat /proc/cpuinfo | grep processor | wc -l`
+#new_json="$(jq '.resource_limits.user_space_size = "4GB" |
```

Comment:
If these configs are not needed, can we remove them?

Reply:
yes, but for some multi-core/multi-socket servers , the default configs does not work properly， the commented out configs is for that， it can make users  to test more efficient